Azure-Samples/media-services-v3-java
# LiveEventWithDVR and EncodingWithMESPredefinedPreset samples

## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
mvn compile exec:java
```

* Test the code

Please see README.md
```

## What to Check
Verify that the following are valid
* ...

## Other Information
